caffe2/utils/CMakeLists.txt

```diff
@@ -1,15 +1,8 @@
-# TODO: Add ThreadPoolXNNPACK.cc when XNNPACK integration is updated
-# to pass the actual threadpool ptr instead of nullptr.
 if(INTERN_BUILD_MOBILE AND NOT BUILD_CAFFE2_MOBILE)
-  add_definitions(-DUSE_INTERNAL_THREADPOOL_IMPL)
   list(APPEND Caffe2_CPU_SRCS
     utils/string_utils.cc
-    utils/threadpool/pthreadpool.cc
-    utils/threadpool/pthreadpool_impl.cc
-    utils/threadpool/pthreadpool_new_if_impl.c
+    utils/threadpool/pthreadpool-cpp.cc
     utils/threadpool/ThreadPool.cc
```

Comment:
Do we need ThreadPool.cc

Reply:
Yes, only because it's got a single function we use, namely getDefaultNumThreads(). If I had moved that function out of ThreadPool.cc and into a common separate file we wouldn't need that file anymore.